index.bs

```diff
@@ -2084,6 +2144,42 @@ operation has no [=authenticator attachment modality=] selection option, so the
 credential|credentials=]. The [=client=] and user will then use whichever is available and convenient at the time.
 
 
+### Resident Key Requirement Enumeration (enum <dfn enum>ResidentKeyRequirement</dfn>) ### {#residentKeyRequirement}
+
+<xmp class="idl">
+    enum ResidentKeyRequirement {
+        "discouraged",
+        "preferred",
+        "required"
+    };
+</xmp>
+
+This enumeration's values describe the [=[RP]=]'s requirements for [=resident credentials=]:
+
+<div dfn-type="enum-value" dfn-for="ResidentKeyRequirement">
+    :   <dfn>discouraged</dfn>
+    ::  This value indicates the client, when communicating with an authenticator, SHOULD NOT instruct the authenticator
+        to create a [=resident credential=]. The authenticator SHOULD create a [=non-resident credential=], however either
+        a [=resident credential=] or [=non-resident credential=] will be accepted by the [=[RP]=].
+
+    :   <dfn>preferred</dfn>
+    ::  This value indicates the client, when communicating with an authenticator, SHOULD instruct the authenticator
+        to create a [=resident credential=]. If the authenticator is incapable of [=resident credentials=], 
+        the client MAY use the authenticator to create a [=non-resident credential=].
+
+    :   <dfn>required</dfn>
+    ::  This value indicates the client MUST use an authenticator capable of [=resident credentials=]. 
+        The authenticator MUST create a [=resident credential=] and MUST return an error if it cannot.
+</div>
+
+Note: The [=[RP]=] can use a combination of the value provided for |options|.{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{residentKey}} 
+and the [[#sctn-authenticator-credential-properties-extension]] [=registration extension=]'s return value to determine 
```

Comment:
strongly suggest s/ [[#sctn-authenticator-credential-properties-extension]] / [=credProps=]  /   otherwise it'll read quite oddly with a section number and full title in there....

Reply:
This has been fixed.